.travis.yml

```diff
@@ -26,4 +31,29 @@ env:
     - TEST_DIR=wallet
     - RUST_TEST_THREADS=1 TEST_DIR=grin
 
+after_success: |
+  pwd &&
+  echo "HERE" &&
+  wget https://github.com/percytheprefect/kcov/archive/master.tar.gz &&
```

Comment:
Why did you need your own fork? Any other solution (like depending on some official release)?

Reply:
The guide has this line: 

> wget https://github.com/SimonKagstrom/kcov/archive/master.tar.gz

I think we want to be able to update kcov at will instead of waiting on `SimonKagstrom` to do it. If you want to fork the repo so that we can use your fork as our reference repository that is fine too.